src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.msbuild

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Part of the reason for the System.Net.Http.WinHttpHandler.msbuild file is so that the other binary, System.Net.Http.dll, can include the sources of WinHttpHandler into its binary (needed to solve circular dependency issue as per @ericstj ).  So, I needed to have a common file (.msbuild) so that the two projects could include the same files, rather than duplicating the file list.